c7n/policy.py

```diff
@@ -307,6 +304,27 @@ def get_logs(self, start, end):
             end,
         )
 
+    def is_runnable(self):
+        now = datetime.now().replace(tzinfo=self.policy.tz)
```

Comment:
i'm not sure this is quite right, it just adds target tz to the current server time, its not converting the current server time to the target tz

```python
>>> datetime.now().replace(tzinfo=tzutc())
datetime.datetime(2018, 7, 6, 13, 7, 41, 483117, tzinfo=tzutc())
>>> datetime.now()
datetime.datetime(2018, 7, 6, 13, 7, 46, 199374)
```

Reply:
i think you want this form instead

```
>>> datetime.now(self.policy.tz)
datetime.datetime(2018, 7, 6, 17, 10, 39, 751654, tzinfo=tzutc())
```